### PR TITLE
16.0 add connector search engine jsonifier

### PR DESCRIPTION
**Pull request by sebastienbeau**
Add connector_search_engine_jsonifier as jsonifier have been removed from base dependency of connector_search_engine see commit : https://github.com/OCA/search-engine/pull/134/commits/8beee1bf92936bf0b3b6a997f4b3965ceafc53c4

### CODE REVIEW

**Comment by sebastienbeau** [comment]
@hparfr should be ready

File: connector_search_engine_serializer_ir_export/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Copyright 2023 Akretion (https://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Connector Search Engine Jsonifier",
```
Comment (hparfr) [inline connector_search_engine_serializer_ir_export/__manifest__.py]:
```suggestion
    "name": "Connector Search Engine Ir Export",
```

File: connector_search_engine_serializer_ir_export/models/se_index.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    @api.depends("serializer_type", "model_id")
+    def _compute_exporter_id(self):
+        for record in self:
+            # reset exporter_id if needed
+            if (
+                record.serializer_type == "jsonifier"
+                and record.exporter_id
+                and record.exporter_id.resource != record.model_id.model
+                or record.serializer_type != "jsonifier"
```
Comment (hparfr) [inline connector_search_engine_serializer_ir_export/models/se_index.py:40]: not really pythonic

**Review by hparfr — CHANGES_REQUESTED** [review]

**Review by hparfr — APPROVED** [review]

**Review by hparfr — APPROVED** [review]

**Pull request merged**